lib/resources.js

```diff
@@ -620,8 +621,11 @@ module.exports = function (app, _private = null) {
 
   // Conduct a search across resources:
   app.resources.search = function (params, opts, request) {
+    app.logger.debug('Unparsed params: ', params)
     params = parseSearchParams(params)
 
+    app.logger.debug('Parsed params: ', params)
```

Comment:
Still need these debugs?

Reply:
I'd leave them in myself, but can remove them if that seems better